### ISSUE TITLE
Mesh display mode should be Flat Lines by default

### ISSUE THREAD

**Issue by FEA-eng** [FreeCAD/FreeCAD#13847]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

Currently, meshes (created in the Mesh wb or imported from STL) have the Display Mode set to Shaded by default. IMO Flat Lines would make more sense as one usually wants to see the division into triangles.

### Full version info

```shell
OS: Windows 10 build 19045
Word size of FreeCAD: 64-bit
Version: 0.22.0dev.37100 (Git)
Build type: Release
Branch: main
```

### Subproject(s) affected?

Mesh

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by luzpaz** [comment]
@maxwxyz should the DWG weigh-in on this ?

**Comment by maxwxyz** [comment]
I think it is just that the flat lines draw style in the mesh view provider does not highlight the triangulated edges currently. It should not change default draw style but the flat lines should be supported in mesh.

**Comment by luzpaz** [comment]
> but the flat lines should be supported in mesh.

they are supported, right? So, in that case you don't agree with the premise of the ticket ?

**Comment by maxwxyz** [comment]
I don't think there is a special handling for that draw mode in mesh.

**Comment by FEA-eng** [comment]
Yeah, that can be the case too. Maybe the default draw style should just be updated to show these edges. It's usually important to see them.
